packages/x-components/src/x-modules/history-queries/components/history-queries.vue

```diff
@@ -17,6 +17,7 @@
           :suggestion="baseScope.suggestion"
           data-test="history-query-item"
           class="x-history-queries__item"
+          suggestionClass="x-suggestion"
```

Comment:
We should take the chance and implement the dynamic prop for the `suggestionClass` like we have in the `HistoryQuery` in the `NextQuery`, `PopularSearch` and `QuerySuggestion` components too

Reply:
Yep, that's a good point, I will include them 👌

Reply:
Looking at this better, you mean to implement the `suggestionItemClass` in the `NextQueries`, `PopularSearches` and `QuerySuggestions` components, right?
The components that you named have the `BaseSuggestion` as the first child, so they don't need a prop for it.